### PR TITLE
[26] Django 1.8

### LINKED ISSUE #26: Django 1.9

### PR DESCRIPTION
**Pull request by dpoirier**
Upgrade from Django 1.7 (now out of support) to
Django 1.8 (Long Term Support)

Fixes #26

### CODE REVIEW

File: reports/test_filters.py [hunk 1]
```diff
@@ -17,9 +17,11 @@ class ReportFilterMixin(object):
```
Comment (vkurup) [inline reports/test_filters.py:17]: Is there a reason that this file is outside the `tests` directory?

File: reports/tests/test_views.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class TestReportList(TestCase):
     template_name = 'reports/reports_list.html'
 
     def setUp(self):
-        super(TestReportList, self).setUp()
+        # super(TestReportList, self).setUp()
```
Comment (vkurup) [inline reports/tests/test_views.py]: This can be uncommented, I think.

**Comment by vkurup** [comment]
A few 1.9 deprecation warnings now pop up. They don't need to be fixed in this PR, but some of them seem pretty simple to fix, so maybe you want to:

```
/home/vkurup/dev/CTS/catalog/utils.py:7: RemovedInDjango19Warning: The django.forms.util module has been renamed. Use django.forms.utils instead.
  from django.forms.util import ErrorDict

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/settings.py:22: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils import importlib, six

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/serializers.py:19: RemovedInDjango19Warning: django.contrib.contenttypes.generic is deprecated and will be removed in Django 1.9. Its contents have been moved to the fields, forms, and admin submodules of django.contrib.contenttypes.
  from django.contrib.contenttypes.generic import GenericForeignKey

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/serializers.py:154: RemovedInDjango19Warning: SortedDict is deprecated and will be removed in Django 1.9.
  return SortedDict(fields)

/home/vkurup/dev/CTS/catalog/utils.py:7: RemovedInDjango19Warning: The django.forms.util module has been renamed. Use django.forms.utils instead.
  from django.forms.util import ErrorDict

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/settings.py:22: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils import importlib, six

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/serializers.py:19: RemovedInDjango19Warning: django.contrib.contenttypes.generic is deprecated and will be removed in Django 1.9. Its contents have been moved to the fields, forms, and admin submodules of django.contrib.contenttypes.
  from django.contrib.contenttypes.generic import GenericForeignKey

/home/vkurup/.virtualenvs/cts/local/lib/python2.7/site-packages/rest_framework/serializers.py:154: RemovedInDjango19Warning: SortedDict is deprecated and will be removed in Django 1.9.
  return SortedDict(fields)
```

Looks reasonable to me :cactus:

File: reports/test_filters.py (hunk 1 above)
Comment (dpoirier) [inline reports/test_filters.py:17]: I have no idea why this file isn't in the `tests` directory. I'll move it there.

**Comment by dpoirier** [comment]
Three of the Django 1.9 deprecation warnings are in Django Rest Framework. They would probably go away if we upgraded it, but upgrading DRF often requires other changes, so I'll pass on that for this pull request.

**Comment by dpoirier** [comment]
This is deployed to staging.

**Pull request merged**